pkgs/development/tools/nodehun/default.nix

```diff
@@ -0,0 +1,49 @@
+{ buildNpmPackage
+, fetchFromGitHub
+, lib
+, nodePackages
+, nodejs
+, python3
+}:
+
+buildNpmPackage rec {
```

Comment:
Can we use the `finalAttrs` pattern with `buildNpmPackage` ?

Let me share a couple of useful links regarding this:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Reply:
The `rec` wasn't needed so I just removed it. Thanks for the review!